sdk/python/pyproject.toml

```diff
@@ -12,16 +12,18 @@ authors = ["Cyral <us@cyral.org>"]
 python = "^3.5"
 boto3 = "^1.14.10"
 grpcio = "^1.30.0"
-psycopg2 = "^2.8.5"
 grpcio-tools = "^1.30.0"
-
+psycopg2 = { version = "^2.8.5", optional = true}
```

Comment:
It looks like for some reason the tests can't find the `psycopg2` library this way.

Reply:
good catch! needed to update the `Dockerfile` to install them since they are now optional.